test/js-native-api/test_exception/test_exception.c

```diff
@@ -38,6 +54,19 @@ static napi_value allowException(napi_env env, napi_callback_info info) {
   return NULL;
 }
 
+static napi_value constructAllowException(napi_env env, napi_callback_info info) {
+  size_t argc = 1;
+  napi_value args[1];
+  NODE_API_CALL(env, napi_get_cb_info(env, info, &argc, args, NULL, NULL));
+
+  napi_value result;
+  napi_new_instance(env, args[0], 0, 0, &result);
+  // Ignore status and check napi_is_exception_pending() instead.
+
```

Comment:
This comment looks a bit misplaced.  Am I correct?
```suggestion

  // Ignore status and check napi_is_exception_pending() instead.
```

Reply:
Eh, maybe not. The comment is actually for the line above, which means that the omitting of the checks on the returning status is intentional.

Reply:
Ah, I see.  Perhaps it would be better to put the comment _above_ the call (and the variable declaration)?  If this is a convention in the codebase no problem then.